### PR TITLE
aws-session-manager-plugin: Add version 1.2.54.0

### LINKED ISSUE #6215: [Request] Add AWS Session Manager Plugin
Please add AWS Session Manager Plugin to this bucket.

Home page is [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html). And Latest download  is [here](https://s3.amazonaws.com/session-manager-downloads/plugin/latest/windows/SessionManagerPluginSetup.exe).

Each version seems to be able to download form https://s3.amazonaws.com/session-manager-downloads/plugin/${version}/windows/SessionManagerPluginSetup.exe

### PR DESCRIPTION
**Pull request by hidekatsu-izuno**
I add a new bucket for  a AWS-CLI session manager plugin.

I hope you can merge this pull request.

Closes #6215

### CODE REVIEW

File: bucket/aws-session-manager-plugin.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "version": "1.2.54.0",
+    "description": "This plugin helps you to use the AWS Command Line Interface (AWS CLI) to start and end sessions to your managed instances",
+    "homepage": "https://github.com/aws/session-manager-plugin",
+    "license": "Apache-2.0",
+    "url": "https://s3.amazonaws.com/session-manager-downloads/plugin/1.2.54.0/windows/SessionManagerPluginSetup.exe",
+    "hash": "91f578f1c30dc4b0ab3af642ef5c9c20586e3dd436775a9ea7c77fa4ed734813",
+    "bin": "bin\\session-manager-plugin.exe",
+    "architecture": {
+        "32bit": {
+            "installer": {
```
Comment (pratikpc) [inline bucket/aws-session-manager-plugin.json:11]: Would it work without the installer script?

Comment (hidekatsu-izuno) [reply bucket/aws-session-manager-plugin.json:11]: I tested it and it works fine. The install has a problem that it cannot specify the installation directory. Therefore, you have to extract it.

**Pull request merged**